README.md

```diff
@@ -108,20 +108,24 @@ When running `modelCheckingTest(),` Lincheck not only detects a bug but also pro
 
 ```text
 = Invalid execution results =
-Init part:
-[addLast(4): void]
-Parallel part:
-| pollFirst(): 4 | addFirst(-4): void       |
-|                | peekLast():   4    [-,1] |
+| -------------------------------------- |
+|     Thread 1     |      Thread 2       |
+| -------------------------------------- |
+| addLast(4): void |                     |
```

Comment:
Are you sure that `addLast(4)` is generated? The arguments should be smaller.

Reply:
This is the same scenario as it was before this PR. I just changed the format of the output to a new one.